polytope/polytope.py

```diff
@@ -171,8 +170,8 @@ def __str__(self):
             spacer = ' |    '
 
             if mid_ind > 1:
-                output += '\n  '.join([A_rows[k]+spacer+b_rows[k] \
-                                        for k in xrange(mid_ind)]) + '\n'
+                output += '\n  '.join([A_rows[k]+spacer+b_rows[k]
+                                      for k in xrange(mid_ind)]) + '\n'
```

Comment:
my preference here is that the `for` is indented to show that it is part of the list comprehension. So, this would be:
```python
                output += '\n  '.join([A_rows[k]+spacer+b_rows[k]
                                       for k in xrange(mid_ind)]) + '\n'
```